src/server/sams/api/consume/assets.py

```diff
@@ -50,6 +53,30 @@ def download_binary(asset_id):
     return response
 
 
+@assets_bp.route('/consume/assets/compressed_binary/<asset_ids>', methods=['GET'])
+def download_compressed_binary(asset_ids):
+    """
+    Uses asset_ids and returns the compressed
+    asset binaries zip
+    """
+    asset_ids = ast.literal_eval(asset_ids) if asset_ids else None
```

Comment:
If `asset_ids` is not provided then this is a client error. Might be a good idea to raise a `SamsAssetErrors` error here instead.